PORTING.md

```diff
@@ -302,4 +302,25 @@ Using [`CRYPTO_BUFFER`](https://commondatastorage.googleapis.com/chromium-boring
 
 ### Asynchronous and opaque private keys
 
-OpenSSL offers the ENGINE API for implementing opaque private keys (i.e. private keys where software only has oracle access because the secrets are held in special hardware or on another machine). While the ENGINE API has been mostly removed from BoringSSL, it is still possible to support opaque keys in this way. However, when using such keys with TLS and BoringSSL, you should strongly prefer using `SSL_PRIVATE_KEY_METHOD` via `SSL[_CTX]_set_private_key_method`. This allows a handshake to be suspended while the private operation is in progress. It also supports more forms of opaque key as it exposes higher-level information about the operation to be performed.
\ No newline at end of file
+OpenSSL offers the ENGINE API for implementing opaque private keys (i.e. private keys where software only has oracle access because the secrets are held in special hardware or on another machine). While the ENGINE API has been mostly removed from BoringSSL, it is still possible to support opaque keys in this way. However, when using such keys with TLS and BoringSSL, you should strongly prefer using `SSL_PRIVATE_KEY_METHOD` via `SSL[_CTX]_set_private_key_method`. This allows a handshake to be suspended while the private operation is in progress. It also supports more forms of opaque key as it exposes higher-level information about the operation to be performed.
+
+## X.509 Certificate Auto-Chaining Disabled by Default
+TLS client or server certificates are loaded using `SSL_CTX_use_certificate_chain_file` or
+`SSL_use_certificate_chain_file`. The file must contain the leaf certificate, and may be followed by one or
+more CA certificates (chain certificates) used to establish the authenticity of the leaf certificate.
```

Comment:
Nit: Awesome description, but just wanted to call out that this could technically also be set up with `SSL(_CTX)_use_certificate` and calling `X509_STORE_add_cert` against the cert store of `SSL(_CTX)`. The wording seems like the discrepancy is a specificity with the two APIs, but it could actually be encountered with any relevant cert/cert store set up in `SSL(_CTX)`.

This test has an example: https://github.com/aws/aws-lc/blob/5cdc082bd86ffb4539e53444d7806a128ace77f3/ssl/ssl_test.cc#L5830-L5888

Reply:
I'll update to include that as an alternative.